### DISCUSSION TITLE
V5 build won't power on with cartridge slot board attached

### DISCUSSION THREAD

**Discussion by Veridor** [sanni/cartreader#637] [Need help]
I just finished a v5 build, and while the GBA slo works fine, when I attach the 6 slot cartridge reader board, the reader no longer powers on. I've checked for solder bridges, and haven't found any so far. Is there something else I can check?

**Comment by Veridor** [comment]
Pictures of my cart slot board

![68F49817-EA19-415A-A4A0-DC7DCA9D2192](https://user-images.githubusercontent.com/1075624/203746298-a48027af-eae5-4d05-a519-339426962b22.jpeg)
![2C55BB17-699D-463E-9A74-F8DADAFDA463](https://user-images.githubusercontent.com/1075624/203746313-80bfdfaa-5de4-47db-b96d-47834cc3a777.jpeg)
![85ADEFEA-94BE-4D4B-B55B-05B1815AA590](https://user-images.githubusercontent.com/1075624/203746320-9b877a5a-ecf2-47a5-b157-dc3669db4ac7.jpeg)

**Reply by Cephiros** [reply]
Nothing jumps out, but test the nothing jumps out, but test vcc and gnd continuity. If they are shorted, i would make sure there isnt a break somewhere in the n64 cable.

**Reply by Veridor** [reply]
> Nothing jumps out, but test the nothing jumps out, but test vcc and gnd continuity. If they are shorted, i would make sure there isnt a break somewhere in the n64 cable.

Thanks. VCC and GND are indeed shorting. I tried removing the N64 cable entirely but the short remains. So I’ve got a short somehow somewhere else on this board.

**Comment by splash5** [comment]
The "legs" of SNES and N64 slots looks longer then others, maybe you can try cut them short (as high as other slots)

**Reply by Veridor** [reply]
Thanks. I trimmed those but still getting a short between VCC and GND like the above commenter said. Thanks for the help, though.

**Comment by Veridor** [comment]
Turns out the shirt was caused by two pins inside the NES cart slot touching. Had to nudge them apart. 

![A4AA0257-8A51-4714-BEBE-202D543F5F61](https://user-images.githubusercontent.com/1075624/203878975-160b9ca7-4b18-4680-8594-b1368ed41983.jpeg)

**Reply by Cephiros** [reply]
Funny, i had considered telling you to out a cart in each slot and test again, but it felt far fetched. Guess it wasnt as far fetched as i thought
